LayoutTests/mathml/presentation/foreign-element-in-annotation-xml-expected.txt

```diff
@@ -35,9 +35,10 @@ h6
 
 blockquote
 
+ERROR
 
 svg
 
 mathml
 
-
```

Comment:
Can we update `foreign-element-in-annotation-xml.html` to use something else than "ERROR"? The comments should also be updated since `<annotation-xml>` allows more children to create renderers.

Ideally the test should be extended too but maybe the HTML parser will forbid some constructions and one will have to create relevant trees using DOM operations instead...